### PR TITLE
Fix exception message when invoking ResultSet.getDate on TIME or TIMESTAMP

### PR DESCRIPTION
**Pull request by jetsasank**
This is a fix for issue : [5313](https://github.com/prestosql/presto/issues/5313) .

### CODE REVIEW

**Review by Praveen2112 — APPROVED** [review]

File: presto-jdbc/src/main/java/io/prestosql/jdbc/AbstractPrestoResultSet.java [hunk 1]
```diff
@@ -279,7 +279,7 @@ private Date getDate(int columnIndex, DateTimeZone localTimeZone)
             return new Date(calendar.getTimeInMillis());
         }
         catch (IllegalArgumentException e) {
-            throw new SQLException("Invalid date from server: " + value, e);
+            throw new SQLException("Expected value to be a date but is : " + value);
```
Comment (findepi) [inline presto-jdbc/src/main/java/io/prestosql/jdbc/AbstractPrestoResultSet.java:282]: There is redundant space before the colon:

```suggestion
            throw new SQLException("Expected value to be a date but is: " + value);
```

File: presto-jdbc/src/test/java/io/prestosql/jdbc/BaseTestJdbcResultSet.java [hunk 2]
```diff
@@ -170,10 +170,10 @@ public void testTime()
                 assertEquals(rs.getObject(column, Time.class), Time.valueOf(LocalTime.of(9, 39, 5)));
                 assertThatThrownBy(() -> rs.getDate(column))
                         .isInstanceOf(SQLException.class)
-                        .hasMessage("Invalid date from server: 09:39:05.000"); // TODO (https://github.com/prestosql/presto/issues/5313) wrong message
+                        .hasMessage("Expected value to be a date but is : 09:39:05.000");
                 assertEquals(rs.getTime(column), Time.valueOf(LocalTime.of(9, 39, 5)));
                 assertThatThrownBy(() -> rs.getTimestamp(column))
-                        .isInstanceOf(IllegalArgumentException.class) // TODO (https://github.com/prestosql/presto/issues/5315) SQLException
```
Comment (findepi) [inline presto-jdbc/src/test/java/io/prestosql/jdbc/BaseTestJdbcResultSet.java:176]: This PR is about `.getDate`'s exception _message_ and the linked issue is about `.getTimestamp`'s exception _type_.

Why the TODO is removed? I guess it was not intentional, please revert.

**Review by findepi — COMMENTED** [review]
Thanks for working on this!

minor comments

**Comment by jetsasank** [comment]
> Thanks for working on this!
> 
> minor comments

Fixed them both. Thank you

**Comment by findepi** [comment]
Merged (52b708682d6b377a315dabdb081313a1f225548c), thanks!

**Pull request closed without merging**